components/function-controller/Makefile

```diff
@@ -81,16 +81,14 @@ ifeq ($(shell uname -s),Darwin)
 	sedbackupflag +=
 endif
 
-deploy: manifests install bin/kustomize
-	@echo "updating kustomize image patch file for manager resource"
-	cp ./config/default/manager_image_patch.yaml{,.bak}
-	sed $(sedbackupflag)'' -e 's@image: .*@image: $(IMG)@' ./config/default/manager_image_patch.yaml
-	bin/kustomize build config/default | kubectl apply -f -
-	mv ./config/default/manager_image_patch.yaml{.bak,}
+deploy: manifests install bin/kustomize ko
+	bin/kustomize build config/default | ko apply -f -
```

Comment:
Why not just using `ko apply` directly? Do we actually want/need kustomize?

Reply:
kustomize adds some additional fields to the yamls, we could either use kustomize, or add all the missing configuration to the yamls